testing/kuttl/e2e/pgbackrest-init/README.md

```diff
@@ -0,0 +1,6 @@
+### pgBackRest Init test
+
+* 00: Create a cluster with two PVC repos and set up for manual backups to go to the second; verify that the PVCs exist and that the backup job completed successfully
+* 01: Run pgbackrest-initialization.sh, which checks that the status matches the expected status of `mixed` (becaue the second repo in the repo list has not yet been pushed to)
```

Comment:
:pick: 
```suggestion
* 01: Run pgbackrest-initialization.sh, which checks that the status matches the expected status of `mixed` (because the second repo in the repo list has not yet been pushed to)
```